### PR TITLE
Runner update

### LINKED ISSUE #1: Update to Wollok 1.3

### PR DESCRIPTION
**Pull request by flbulgarelli**
Updates this runner to 

  * wollok 1.3 beta
  * mumukit 1.1
  * escualo 2

Fixes #1

### CODE REVIEW

**Comment by flbulgarelli** [comment]
@npasserini @umanskyariel this updates Wollok support. 

Are there significative changes from 1.3-beta to 1.3? Should we update to 1.3 or 1.4?

File: config.ru [hunk 1]
```diff
@@ -1,8 +1,4 @@
-#\ -s puma
-require 'rubygems'
-require 'bundler'
-
-Bundler.require
+#\-s puma
```
Comment (faloi) [inline config.ru]: Why is this necessary?

**Comment by faloi** [comment]
It's ok for me, merge at will.

**Comment by npasserini** [comment]
1.4

File: config.ru (hunk 1 above)
Comment (flbulgarelli) [inline config.ru]: Because puma, not like thin, does not run by default if present

**Comment by flbulgarelli** [comment]
@npasserini OK, but it there a 1.4 SDK already ready to deploy? 

I am not finding 1.3 stable nor 1.4-SNAPSHOT here: http://download.uqbar.org/wollok/sdk/

**Comment by npasserini** [comment]
Javi?

**Comment by npasserini** [comment]
This should be the link

http://download.uqbar.org/wollok/sdk/dev/org.uqbar.project.wollok.launch-1.3.0-SNAPSHOT-wdk.zip

Even if the name says 1.3.0 it is actually the build from "dev" done last
week after merging 1.4 branch.
So, it is the upcoming 1.4

**Comment by flbulgarelli** [comment]
Excellent! Thanks
